doc/source/cluster/kubernetes/k8s-ecosystem/yunikorn.md

```diff
@@ -0,0 +1,190 @@
+
+:::
+
+Because the queue has a capacity of 4 CPU and 6Gi of RAM, this resource should schedule successfully without any issues.
+
+```shell
+$ kubectl get pods
+
+NAME                                  READY   STATUS    RESTARTS   AGE
+test-yunikorn-0-head-98fmp            1/1     Running   0          67s
+test-yunikorn-0-worker-worker-42tgg   1/1     Running   0          67s
+test-yunikorn-0-worker-worker-467mn   1/1     Running   0          67s
+```
+
+You can verify it by checking the [Apache YuniKorn dashboard](https://yunikorn.apache.org/docs/#access-the-web-ui).
+
+```shell
+kubectl port-forward svc/yunikorn-service 9889:9889 -n yunikorn
+```
+
+And then go to http://localhost:9889/#/applications to see the running applications.
+
+![Apache YuniKorn dashboard](../images/yunikorn-dashboard-apps-running.png)
+
+Next, add an additional RayCluster with the same configuration of head and worker nodes, but with a different name:
+
+```shell
+# Replace the name with `test-yunikorn-1`
+sed 's/test-yunikorn-0/test-yunikorn-1/' ray-cluster.yunikorn-scheduler.yaml | kubectl apply -f-
+```
+
+Now all the Pods for `test-yunikorn-1` are in the `Pending` state:
+
+```shell
+$ kubectl get pods
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+test-yunikorn-0-head-98fmp                1/1     Running   0          4m22s
+test-yunikorn-0-worker-worker-42tgg       1/1     Running   0          4m22s
+test-yunikorn-0-worker-worker-467mn       1/1     Running   0          4m22s
+test-yunikorn-1-head-xl2r5                0/1     Pending   0          71s
+test-yunikorn-1-worker-worker-l6ttz       0/1     Pending   0          71s
+test-yunikorn-1-worker-worker-vjsts       0/1     Pending   0          71s
+tg-test-yunikorn-1-headgroup-vgzvoot0dh   0/1     Pending   0          69s
+tg-test-yunikorn-1-worker-eyti2bn2jv      1/1     Running   0          69s
+tg-test-yunikorn-1-worker-k8it0x6s73      0/1     Pending   0          69s
+```
+
+Those Pods that prefixed with `tg-` are created by Apache YuniKorn for gang scheduling purpose.
+
+Go to http://localhost:9889/#/applications and then you can see `test-yunikorn-1` is in the `Accepted` state but not running yet:
+
+![Apache YuniKorn dashboard](../images/yunikorn-dashboard-apps-pending.png)
+
+Because the new cluster requires more CPU and RAM than the queue allows, even though one of the Pods would fit in the remaining 1 CPU and 2Gi of RAM, none of the cluster's Pods are placed until there is enough room for all the Pods. Without using Apache YuniKorn for gang scheduling in this way, one of the Pods would ordinarily be placed, leading to the cluster being partially allocated.
+
+Delete the first RayCluster to free up resources in the queue:
+
+```shell
+kubectl delete raycluster test-yunikorn-0
+```
+
+Now all the Pods for the second cluster changed to the `Running` state, because enough resources are now available to schedule the entire set of Pods:
```

Comment:
```suggestion
Now all the Pods for the second cluster change to the `Running` state, because enough resources are now available to schedule the entire set of Pods:
```